### DISCUSSION TITLE
How to change default title "Browser Wallet" of injected connector?

### DISCUSSION THREAD

**Discussion by hiunkim** [WalletConnect/walletconnect-monorepo#4246] [Developer Support]
Hello all! Could you please advise is there a way to change default title "Browser Wallet" of injected connector?
Im able to change logo by using connectorImages option

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/11682893/3b8e3209-a66b-4f7b-891e-2fcea69e1bf8)

**Comment by glitch-txs** [comment]
This is currently not possible, could you open a GitHub issue for this?
https://github.com/WalletConnect/web3modal/issues

**Comment by DanilKasianenko** [comment]
I was open new issue for this problem
https://github.com/WalletConnect/web3modal/issues/1995
